src/pages/contact_us.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../../assets/css/output.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+    integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/open-sans" rel="stylesheet">
+</head>
+
+<body>
+    <div id="header"></div>
+
+    <div class="container mx-auto max-w-7xl mb-24 text-sm px-2.5">
+        <nav class="flex my-4">
+            <ol class="inline-flex text-gray-400 font-bold space-x-2">
+                <li>
+                    <a href="./index.html" class="hover:text-primary-color" data-i18n="breadcrumb.home">Trang chủ</a>
```

Comment:
```suggestion
                    <a href="./index.html" class="hover:text-primary-color" data-i18n="breadcrumb.home"></a>
```
E bỏ những text bên trong đi nha